code-reviews/example1/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
Have you considered maybe that a meta tag is missing to make it more responsive?